backend/app/store/service/service.go

```diff
@@ -200,6 +200,30 @@ func (s *DataStore) DeleteUserDetail(siteID string, userID string, detail engine
 	})
 }
 
+// ResubmitStagingImages retrieves timestamp of the oldest image in staging and
+// calls s.submitImages on all comments newer than it
+func (s *DataStore) ResubmitStagingImages(sites []string) error {
+	info, err := s.ImageService.Info()
+	if err != nil {
+		return err
+	}
+	ts := info.FirstStagingImageTS
+	if ts.IsZero() {
+		return nil
+	}
+	for _, site := range sites {
+		locator := store.Locator{SiteID: site}
+		comments, err := s.FindSince(locator, "time", store.User{}, ts)
+		if err != nil {
```

Comment:
as this operation runs for different sites it makes sense not to terminate on the first error but collect in multierror and continute.And return collected error(s) after this